fs2/src/main/scala/laserdisc/fs2/RedisConnection.scala

```diff
@@ -48,22 +48,34 @@ object RedisConnection {
 
       def framing: Pipe[F, Byte, CompleteFrame] = {
 
-        def go(stream: Stream[F, Byte], previous: Frame): Pull[F, CompleteFrame, Unit] =
+        def produceMultipleRight[A](xs: List[A]): Pull[F, A, Unit] =
```

Comment:
I'm not sure if there is a better way in fs2 for doing this. Any idea ?

Reply:
you want to emit the contents of `xs` (in reverse order) in here, right?
Not sure about perf but... what about `Pull.seq(xs.reverse)`? `Pull.seq` will actually convert it to `Vector` 😢

Reply:
Sorry, I conflated two things: `Pull.seq` => `Pull.outputChunk(Chunk.seq(xs))` instead